src/require.jl

```diff
@@ -85,6 +85,15 @@ function __init__()
           NLopt.upper_bounds!(opt, upper_bounds)
           NLopt.maxeval!(opt, maxeval)
 
+          if nstart > 1
+            localopt = opt
+            opt = NLopt.Opt(:G_MLSL_LDS, length(lower_bounds))
```

Comment:
I would expect this to be passed in as the `opt`, it would be better to have a `localopt` kwarg and use that. This also hard codes the name, in the NLopt docs I can see Augmented Lagrangian algorithm is another algorithm that would work this way.